tests/storage/__init__.py

```diff
@@ -53,9 +53,11 @@ def populated_db(tmpdir):
     prior = repository.add_prior(
         "group", "key1", "function", ["paramA", "paramB"], [0.1, 0.2]
     )
+    repository.flush()
     prior1 = repository.add_prior(
         "group", "key2", "function", ["paramA", "paramB"], [0.3, 0.4]
     )
+    repository.flush()
```

Comment:
Note that these flush statements were added because the test relies on `"key1"` being the prior with `id=1`. This is not guaranteed when just adding all three and flushing later, and caused flaky tests,

Reply:
👍